File: data/imported/knowledge/cells/go/merged/snippets/snippet_001.go
func main() {
	wb := cells_foss.NewWorkbook()
	ws := wb.Worksheets[0]

	// Populate a few cells.
	ws.Cells().Set("A1", "Hello")
	ws.Cells().Set("B1", "World")
	ws.Cells().Set("A2", "Answer")
	ws.Cells().Set("B2", 42)

	outPath := "outputfiles/basic.xlsx"
	if err := wb.Save(outPath); err != nil {
		fmt.Printf("Error saving workbook: %v\n", err)
		return
	}
	fmt.Printf("Workbook saved to %s\n", outPath)
}

File: data/imported/knowledge/cells/typescript/merged/snippets/snippet_001.ts
async function testAutoFilter() {
  console.log("Testing auto filter...");

  const workbook = new Workbook();
  const worksheet = workbook.worksheets[0]!;

  worksheet.putValue("A1", "Name");
  worksheet.putValue("B1", "Age");
  worksheet.putValue("C1", "City");

  worksheet.putValue("A2", "Alice");
  worksheet.putValue("B2", "25");
  worksheet.putValue("C2", "New York");

  worksheet.putValue("A3", "Bob");
  worksheet.putValue("B3", "30");
  worksheet.putValue("C3", "London");

  worksheet.putValue("A4", "Charlie");
  worksheet.putValue("B4", "35");
  worksheet.putValue("C4", "Paris");

  worksheet.setAutoFilter("A1:C4");

  await workbook.save("outputfiles/test_auto_filter.xlsx");
  console.log("Saved to outputfiles/test_auto_filter.xlsx");
}